packages/babel-helper-module-imports/src/is-module.ts

```diff
@@ -5,12 +5,5 @@ import type * as t from "@babel/types";
  * A small utility to check if a file qualifies as a module.
  */
 export default function isModule(path: NodePath<t.Program>) {
-  const { sourceType } = path.node;
-  if (sourceType !== "module" && sourceType !== "script") {
-    throw path.buildCodeFrameError(
-      `Unknown sourceType "${sourceType}", cannot transform.`,
-    );
-  }
-
```

Comment:
Why do we need this change?

Reply:
Now babel must be one of the two, unless one uses babal-7-beta and writes a new bug to trigger this.